src/components/picker/helpers/useNewPickerProps.tsx

```diff
@@ -24,11 +27,19 @@ const useNewPickerProps = (props: PickerProps) => {
     onRequestClose: topBarProps?.onCancel
   };
 
+  const showTopBar = (!useDialog || mode === PickerModes.MULTI) && !renderHeader;
+  const overlayTopBarProps = showTopBar ? {testID: `${testID}.modal.topBar`, ...topBarProps} : undefined;
```

Comment:
The testID can be passed from the Picker. 
Why is it called "...modal.topBar' and not "..overlay.topBar"? Are you not passing it to the Dialog as well?

Reply:
This is the testID that used in the PickerItemList, changing it might break some users test's.

Reply:
This is what I see: `testID={`${props.testID}.topBar`}`